main.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const { createServer } = require('./src/createServer.js');
+
+const PORT = process.env.PORT || 8080;
+
+createServer().listen(3000, () => {
```

Comment:
You've already put so much effor in getting PORT from env, please use it here instead of 3000 😄